packages/eslint-plugin/src/configs/recommended.json

```diff
@@ -28,13 +29,15 @@
     "@typescript-eslint/no-non-null-assertion": "error",
     "@typescript-eslint/no-object-literal-type-assertion": "error",
     "@typescript-eslint/no-parameter-properties": "error",
+    "@typescript-eslint/no-require-imports": "error",
     "@typescript-eslint/no-triple-slash-reference": "error",
     "no-unused-vars": "off",
     "@typescript-eslint/no-unused-vars": "warn",
     "@typescript-eslint/no-use-before-define": "error",
     "@typescript-eslint/no-var-requires": "error",
     "@typescript-eslint/prefer-interface": "error",
     "@typescript-eslint/prefer-namespace-keyword": "error",
+    "@typescript-eslint/promise-function-async": "error",
```

Comment:
we should not modify recommended configs without major release, can you remove recommended options from those rules.

Reply:
I see eslint:recommended only changes with major release and sure I will remove changes on recommended.json. But removing the recommended setting in these rules might require to introduce a new "process" to check and enable / disable all the recommended rules before creating a new major. Is this intended?